api/client/gro_client.py

```diff
@@ -181,7 +181,22 @@ def add_single_data_series(self, data_series):
 
 
     def find_data_series(self, **kwargs):
-        """Attempts to find a matching data series for a combination of entities by name.
+        """Find the best possible  data series matching a combination of entities specified by name.
+
+        Example::
+
+            client.find_data_series(item="Corn",
+                                    metric="Futures Open Interest",
+                                    region="United States of America").next()
```

Comment:
`iter.next()` is Python 2

Update to `next(iter)`?

Reply:
done